File: program/convert-centimeters-to-inches/convert-centimeters-to-inches.py
#ISSUE #3412
def centimetersToInch(centimeters):
    return centimeters/2.54

centimeters = 12.7
print(('Input: '+str(centimeters) + " {oneOrMore}").format(oneOrMore="centimeter" if centimeters == 1 else "centimeters"))
result = centimetersToInch(centimeters)
print(('Output: '+str(result) + " {oneOrMore}").format(oneOrMore="inch" if result == 1 else "inches"))

File: program/convert-centimeters-to-inches/ConvertCentimetersToInches.js
/*ISSUE #3407*/
function centimetersToInch(centimeters) {
    return centimeters / 2.54;
  }

  const centimeters = 12.7;
  console.log(`Input: ${centimeters} ${centimeters > 1 ? 'centimeters' : 'centimeter'}`);
  const result = centimetersToInch(centimeters);
  console.log(`Output: ${result} ${result == 1 ? 'inch' : 'inches'}`);